handsontable/src/styles/legacy/handsontable.css

```diff
@@ -1890,8 +1897,8 @@ textarea.HandsontableCopyPaste {
   font-size: 8px;
   height: 8px;
   line-height: 1.1;
-  /* Workaround for IE9 - IE11 */
-  text-decoration: underline;
+  /* Workaround for IE9 - IE11, https://stackoverflow.com/a/21902566, https://stackoverflow.com/a/32120247 */
```

Comment:
Can be safely removed? (not supported browsers)

Reply:
🤞